pydis_site/apps/guides/resources/guides/how-to-write-a-guide.md

```diff
@@ -0,0 +1,60 @@
+Title: How to Write a Guide
+ShortDescription: Learn how to write a guide for this website
+Contributors: ks129
+
+When you are interested about how to write guide for this site (like this), then you can learn about it here.
+PyDis use Markdown files for guides, but these files have some small differences from standard Markdown (like defining HTML IDs and classes).
+
+## [Getting Started](#getting-started){: id="getting-started" }
+First, you have to have a good idea, that match with PyDis theme. We can't accept guides like *How to bake a cake*,
+*How to lose weigth*. These doesn't match with PyDis theme and will be declined. Most of guides theme should be server and Python, but there can be some exceptions, when they are connected with PyDis.
```

Comment:
I don't think this document has to explain any kind of policy, just focus it on the technical details.